packit/config/package_config.py

```diff
@@ -267,7 +267,9 @@ def get_local_package_config(
     cwd = Path.cwd()
 
     if try_local_dir_first and try_local_dir_last:
-        logger.error("Ambiguous usage of try_local_dir_first and try_local_dir_last")
+        logger.error(
+            "Ambiguous usage of `try_local_dir_first` and `try_local_dir_last`."
```

Comment:
```suggestion
            "Ambiguous usage of 'try_local_dir_first' and 'try_local_dir_last'."
```

I remember back-ticks were causing some issues when propagated to a markdown context. Not sure if this ever going to happen to this line, but maybe avoiding them could be a good practice? :thinking: